UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs

```diff
@@ -79,7 +79,7 @@ void IInitialise.Initialise()
 
 	        if (PlayerPrefs.HasKey(PlayerPrefKeys.EnableClassicHoverTooltip) == false)
 	        {
-		        ClassicHoverTooltipToggle(false);
+		        ClassicHoverTooltipToggle(true);
```

Comment:
this is way too odd. What if I see the option in my settings and enable it? If you want to force it enabled, just get rid of the if or something like that.

Reply:
ah, good point. we aren't updating `IsOn` directly so it wouldn't be updated on the user's end in the options menu

Reply:
it's now enabled by default in the options menu as well